File: C++/1045.cpp
#include <stdio.h>
#include <math.h>
int main() {
    double a,b,c;
    scanf("%lf %lf %lf", &a,&b,&c);

    if (((a+b)>c) && ((a+c)>b) && ((b+c)>a) && ((a-b)<c) && ((a-c)<b) && ((b-c)<a)){
    	if(((a*a)==((b*b)+(c*c))) || ((b*b)==((a*a)+(c*c))) || ((c*c)==((a*a)+(b*b))) ) printf("TRIANGULO RETANGULO\n");
    	else{
		
    	if((a*a)>((b*b)+(c*c)) || ((b*b)>((a*a)+(c*c))) || (c*c)>((a*a)+(b*b))) printf("TRIANGULO OBTUSANGULO\n");
    	if((a*a)<((b*b)+(c*c)) && ((b*b)<((a*a)+(c*c))) && (c*c)<((a*a)+(b*b)))printf("TRIANGULO ACUTANGULO\n");
    	if (a==b && a==c) printf("TRIANGULO EQUILATERO\n");
    	
    }
    }
    else printf("NAO FORMA TRIANGULO\n");
    if((a==b && a!=c) || (a==c && a!=b) || (b==c && b!=a) ) printf ("TRIANGULO ISOSCELES\n");
    	
    
   	return 0;
   	
   	
   }

File: Python/1045.py
def bubblesort(arr):
    n = len(arr)
    for i in range(n - 1):
        for j in range(0, n - i - 1):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]


a, b, c = input().split()
a = float(a)
b = float(b)
c = float(c)

arr = [a, b, c]

bubblesort(arr)

if not(((arr[0] + arr[1]) > arr[2]) & ((arr[2] - arr[1]) < arr[0])):
    print(f'NAO FORMA TRIANGULO')
else:
    if (arr[2]*arr[2]) == ((arr[1]*arr[1]) + (arr[0])*arr[0]):
        print(f'TRIANGULO RETANGULO')
    if (arr[2]*arr[2]) > ((arr[1]*arr[1]) + (arr[0])*arr[0]):
        print(f'TRIANGULO OBTUSANGULO')
    if (arr[2]*arr[2]) < ((arr[1]*arr[1]) + (arr[0])*arr[0]):
        print(f'TRIANGULO ACUTANGULO')
    if a == b == c:
        print(f'TRIANGULO EQUILATERO')
    elif (a == b) | (a == c) | (b == c):
        print(f'TRIANGULO ISOSCELES')
